splice_access_api/src/main/java/com/splicemachine/access/configuration/SIConfigurations.java

```diff
@@ -180,6 +181,10 @@
     public static final String DURABILITY = "splice.txn.durability";
     public static final String DEFAULT_DURABILITY = "SYNC";
 
+    // write-write conflict resolution strategy
+    public static final String CONFLICT_RESOLUTION_STRATEGY = "splice.txn.conflictResolutionStrategy";
+    public static final String DEFAULT_CONFLICT_RESOLUTION_STRATEGY = ConflictResolutionStrategy.DEFERRED.toString();
```

Comment:
as discussed, on master we activate the new behavior, but when cherry-picking this PR on 3.0 and 3.1 we keep the old one (`IMMEDIATE`)